chrome/content/scripts/zoteroshortdoi.js

```diff
@@ -1,493 +0,0 @@
-// Startup -- load Zotero and constants
-if (typeof Zotero === 'undefined') {
-    Zotero = {};
-}
-Zotero.ShortDOI = {};
-
-// Preference managers
-
-Zotero.ShortDOI.getPref = function(pref) {
-    return Zotero.Prefs.get('extensions.shortdoi.' + pref, true);
-};
-
-Zotero.ShortDOI.setPref = function(pref, value) {
-    return Zotero.Prefs.set('extensions.shortdoi.' + pref, value, true);
-};
-
-// Startup - initialize plugin
-
-Zotero.ShortDOI.init = function() {
-    Zotero.ShortDOI.resetState("initial");
-
-    // Register the callback in Zotero as an item observer
-    var notifierID = Zotero.Notifier.registerObserver(
-        Zotero.ShortDOI.notifierCallback, ['item']);
-
-    // Unregister callback when the window closes (important to avoid a memory leak)
-    window.addEventListener('unload', function(e) {
-        Zotero.Notifier.unregisterObserver(notifierID);
-    }, false);
-
-};
-
-Zotero.ShortDOI.notifierCallback = {
-    notify: function(event, type, ids, extraData) {
-        if (event == 'add') {
-          switch (Zotero.ShortDOI.getPref("autoretrieve")) {
-            case "short":
-              Zotero.ShortDOI.updateItems(Zotero.Items.get(ids), "short");
-              break;
-            case "long":
-              Zotero.ShortDOI.updateItems(Zotero.Items.get(ids), "long");
-              break;
-            case "verify":
```

Comment:
This check was already buggy, since the autoretrieve setting would be set to `check` and not `verify`.